### ISSUE TITLE
Content section

### ISSUE THREAD

**Issue by Xa6ep** [mnyrop/pagemaster#19]
Hi, I trying to use this plugin with examples from sample_site folder.
It works (Jekyll 4.0.0 isn't), I receive this in generated page:
```
---
pid: dir_imgs_item
gambrel: 'null'
indescribable: Pie. Whoever invented the pie? Here was a great person.
blasphemous: Shunned lurk hideous antiquarian foetid furtive antediluvian. Unmentionable
  accursed gibbous ululate cyclopean. Fainted antiquarian hideous eldritch immemorial
  unnamable dank singular. Ululate daemoniac madness loathsome cat non-euclidean foetid
  swarthy. Noisome manuscript cat nameless gibbous.
furtive: Mathias Buckridge
permalink: "/csv_collection/dir_imgs_item.html"
layout: default
---
```
How can I fill content section? Of course, I can add content in layout, but I feel its wrong way.
```
<div class="wrapper">{{ page.blasphemous }}</div>
```

**Comment by mnyrop** [comment]
Hi @Xa6ep, using the layout is the correct way. 
Also, are you saying the plugin doesn't work at all with Jekyll 4? Has a version 4 been officially released?

**Comment by Xa6ep** [comment]
Yes, 27 days ago it was officially released.

```
> bundle install
Fetching gem metadata from https://rubygems.org/...........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "jekyll":
  In snapshot (Gemfile.lock):
    jekyll (= 4.0.0)

  In Gemfile:
    jekyll (~> 4.0.0) x64-mingw32

    pagemaster x64-mingw32 was resolved to 2.0.1, which depends on
      jekyll (~> 3.0) x64-mingw32

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

> bundle update
Fetching gem metadata from https://rubygems.org/...........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "jekyll":
  In Gemfile:
    jekyll (~> 4.0.0) x64-mingw32

    pagemaster x64-mingw32 was resolved to 2.0.1, which depends on
      jekyll (~> 3.0) x64-mingw32

```

**Comment by mnyrop** [comment]
I'll test this and add a patch in the new release. Thanks for the heads up!

(Moving to standalone issue #20)
